firefox-ios/Client/Frontend/Browser/StartAtHomeHelper.swift

```diff
@@ -44,25 +44,20 @@ class StartAtHomeHelper: FeatureFlaggable {
     /// the user's last activity and whether, based on their settings, Start at Home feature
     /// should perform its function.
     public func shouldStartAtHome() -> Bool {
-        let setting = startAtHomeSetting
-        guard setting != .disabled else { return false }
-
         let lastActiveTimestamp = UserDefaults.standard.object(forKey: "LastActiveTimestamp") as? Date ?? Date()
         let dateComponents = Calendar.current.dateComponents([.hour, .second],
                                                              from: lastActiveTimestamp,
                                                              to: Date())
-        var timeSinceLastActivity = 0
-        var timeToOpenNewHome = 0
-
-        if setting == .afterFourHours {
-            timeSinceLastActivity = dateComponents.hour ?? 0
-            timeToOpenNewHome = 4
-        } else if setting == .always {
-            timeSinceLastActivity = dateComponents.second ?? 0
-            timeToOpenNewHome = 5
+        switch startAtHomeSetting {
+        case .afterFourHours:
+            let hoursSinceLastActive = dateComponents.hour ?? 0
+            return hoursSinceLastActive >= 4
+        case .always:
+            let secondsSinceLastActive = dateComponents.second ?? 0
+            return secondsSinceLastActive >= 5
```

Comment:
Could we add 4 and 5 to a private struct like Constants, so we reference to those number with a constant that has a meaningful name, what do you think ?
```swift
class ... {
private struct Constants {
 ...
 ...
}
}
```

Reply:
Great suggestion! I've moved the hardcoded values for hours and seconds to a Constants struct to make the code more readable and maintainable

Reply:
Thanks very much @bmihai23 😃